specification/sphere/resource-manager/readme.md

```diff
@@ -31,6 +31,11 @@ openapi-subtype: rpaas
 tag: package-2024-04-01
 ```
 
+``` yaml
```

Comment:
Since [Azure.ResourceManager.Sphere/1.0.0](https://www.nuget.org/packages/Azure.ResourceManager.Sphere/1.0.0), we can't disable the flatten for .NET.

Reply:
@live1206 If language owner don't want it for your language, please add ```` yaml !$(LANGUAGENAME)`. Thanks!

Reply:
could the PR initializer help to add this? thanks

Reply:
@live1206 @msyyc has been updated, please review.